time.c

```diff
@@ -4160,13 +4160,16 @@ time_add(const struct time_object *tobj, VALUE torig, VALUE offset, int sign)
 
 /*
  *  call-seq:
- *     time + numeric -> time
+ *    self + addend -> new_time
```

Comment:
Probably better to keep this as `numeric`.

Reply:
Fixed.